etc/calico/confd/conf.d/bird_ipam.toml

```diff
@@ -6,5 +6,6 @@ keys = [
     "/v1/ipam/v4/pool",
     "/bgp/v1/host//NODENAME",
     "/staticroutes",
+    "/rejectcidrs",
```

Comment:
I added a new pseudo-key, `/calico/rejectcidrs` which is used to populate a list of CIDRs we reject.

The template uses `~` to reject all routes within the CIDR, so we don't need to write an entry for every clusterIP.